typescript/sdk/src/ism/HyperlaneIsmFactoryDeployer.ts

```diff
@@ -0,0 +1,47 @@
+import debug from 'debug';
+
+import { HyperlaneContracts } from '../contracts';
+import { HyperlaneDeployer } from '../deploy/HyperlaneDeployer';
+import { MultiProvider } from '../providers/MultiProvider';
+import { ChainMap, ChainName } from '../types';
+
+import { IsmFactoryFactories, ismFactoryFactories } from './contracts';
+
+export type IsmFactoryConfig = any;
+
+export class HyperlaneIsmFactoryDeployer extends HyperlaneDeployer<
+  IsmFactoryConfig,
+  IsmFactoryFactories
+> {
+  constructor(
+    multiProvider: MultiProvider,
+    configMap: ChainMap<IsmFactoryConfig>,
+    factoriesOverride = ismFactoryFactories,
+  ) {
+    super(multiProvider, configMap, factoriesOverride, {
+      logger: debug('hyperlane:IsmFactoryDeployer'),
+    });
+  }
+
+  async deployContracts(
+    chain: ChainName,
+    config: IsmFactoryConfig,
```

Comment:
does the config not do anything?

Reply:
Indeed it does not, there's nothing to config other than which chains to deploy to